### PR TITLE
Update compressx extension

### PR DESCRIPTION
**Pull request by dqhieu**
## Description

- Add new commands: `Compress Files` and `Import Files`

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/compressx/package.json [hunk 1]
```diff
@@ -11,22 +11,29 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "compressVideo",
```
Comment (pernielsentikaer) [inline extensions/compressx/package.json:14]: Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Comment (dqhieu) [reply extensions/compressx/package.json:14]: Reversed. Thanks for the recommendation!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by dqhieu** [comment]
Thank you for the quick review

**Pull request merged**